node/node_syncing.go

```diff
@@ -235,7 +235,9 @@ SyncingLoop:
 				node.Consensus.BlocksNotSynchronized()
 			}
 			node.stateSync.SyncLoop(bc, worker, false)
-			if node.NodeConfig.Role() == nodeconfig.ExplorerNode {
+			// update the consensus and committee information at the end of epoch including explorer node
```

Comment:
This is not enough right? This update happens only after SyncLoop finishes (which means it only happens after it's sync'ed, not during the sync)

Reply:
Right. I changed it into syncLoop, I think now it should be good